integration/test-runner/package.json

```diff
@@ -14,7 +14,6 @@
   },
   "author": "modern-web",
   "homepage": "https://github.com/modernweb-dev/web/tree/master/packages/test-runner-integration-tests",
-  "main": "index.js",
```

Comment:
Why remove this?

Reply:
It's unrelated to the chai changes. I was just getting warnings when running the tests:

```
(node:87634) [DEP0128] DeprecationWarning: Invalid 'main' field in '/src/web/integration/test-runner/package.json' of 'index.js'. Please either fix that or report it to the module author
```